book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    return if @rentals.include?(rental)
+
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Here passing `rental ` to a function not b able to create rental, I think the proper code should be this. Kindly add it.
```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```